### PR TITLE
Full TimeSlider

### PR DESCRIPTION
**Pull request by aherreDev**

### CODE REVIEW

**Comment by aherreDev** [comment]
Frontend changes needed:
- [ ] Timeslider UI
- [ ] Worldmap to support date selection
- [ ] Popup UI to support date selection
- [ ] Define default datapoints that assumes missing data from JSON as `no data`

File: src/components/TimeSlider.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ Date.prototype.toSliderString = function () {
   newDate[2] = year;
   return newDate.join('/');
 };
+Date.prototype.toJsonString = function () {
```
Comment (davojta) [inline src/components/TimeSlider.js:16]: it is not good idea to monkey patch the default window objects

 Could we use something like  as we already have date-fns in the project ?

```
import format from 'date-fns/format'

const getTodayDate = () => {
    const now = Date.now();

    return format(now, 'yyyy-MM-dd');
}
```

Comment (aherreDev) [reply src/components/TimeSlider.js:16]: let me change it

Comment (aherreDev) [reply src/components/TimeSlider.js]: It's done

**Pull request merged**